ntoskrnl/cc/copy.c

```diff
@@ -400,11 +405,40 @@ CcCanIWrite (
         return FALSE;
     }
 
-    /* Otherwise, if there are no deferred writes yet, start the lazy writer */
-    if (IsListEmpty(&CcDeferredWrites))
+#if DBG
+    /* Output to debug log, before handling the lazy write */
+    DPRINT1("Actively deferring write for: %p \"%wZ\"\n", FileObject, &FileObject->FileName);
+    DPRINT1("Because:\n");
```

Comment:
Obviously, empty causes can still happen.
I assume this is caused by `Cc*` and `Mm*` values, which can change at any time, can't they?
Should we capture these values too, to at least have a consistent debug log? Or do we not care about current partly-random result?
And/or should we try to "synchronize" more of this code, if possible?